docs/index.rst

```diff
@@ -88,36 +148,77 @@ The plugin can also play the file in a loop which means it can start again if en
                 It is left to the user to ensure there are no missing values in CSV file. However if the option selected is report. Then plugin will check for NaN's and report error to user. This can serve as a way to check the CSV file for missing values. However the user has to take action on what to do with NaN values. The default action is to ignore them.
                 When error is reported the user must delete the south service and try again with clean CSV file.
 
-  - **'repeatLoop': type: boolean default: false**:
-                Read CSV in a loop i.e. on reaching End Of File, again go back to beginning of the file.
+   - **'postProcessMethod': type: enumeration default: 'continue_playing'**:
+                It is the method to process the CSV file once all rows are ingested.
+                It could be:
+
+                1. continue_playing
```

Comment:
It is not really "continue_playing", it is replaying.

Reply:
Will change that in future.